official/resnet/resnet_run_loop.py

```diff
@@ -266,14 +266,67 @@ def learning_rate_fn(global_step):
                      false_fn=lambda: lr)
     return lr
 
+
+
+  def poly_rate_fn(global_step):
+    """Handles linear scaling rule, gradual warmup, and LR decay.
+
+    The learning rate starts at 0, then it increases linearly per step.  After
+    FLAGS.poly_warmup_epochs, we reach the base learning rate (scaled to account
+    for batch size). The learning rate is then decayed using a polynomial rate
+    decay schedule with power 2.0.
+
+    Args:
+    global_step: the current global_step
+
+    Returns:
+    returns the current learning rate
+    """
+
+    # Learning rate schedule for LARS polynomial schedule
```

Comment:
I think this is different from before. Shouldn't it be:
```
if flags.FLAGS.batch_size < 8192:
  plr = 5.0
  w_epochs = 5
elif flags.FLAGS.batch_size < 16384:
  plr = 10.0
  w_epochs = 5
if flags.FLAGS.batch_size < 32768:
  plr = 25.0
  w_epochs = 5
else:
  plr = 32.0
  w_epochs = 14
```

Reply:
facepalm. fixed.